app/Models/Asset.php

```diff
@@ -105,16 +105,14 @@ public function declinedCheckout(User $declinedBy, $signature)
         'company_id'      => 'integer|nullable',
         'warranty_months' => 'numeric|nullable|digits_between:0,240',
         'physical'        => 'numeric|max:1|nullable',
-        'checkout_date'   => 'date|max:10|min:10|nullable',
-        'checkin_date'    => 'date|max:10|min:10|nullable',
+        'last_checkout'    => 'date|date_format:Y-m-d H:i:s|nullable',
+        'expected_checkin' => 'date|date_format:Y-m-d|nullable',
```

Comment:
Should `date` be removed here and below for `purchase_date` in favor of `date_format:Y-m-d` like in the other models (Accessory, Component, Consumable, etc)?

The results are different for this model:
![Component](https://user-images.githubusercontent.com/1141514/214654556-5b19c3fb-344d-4a48-80f7-f71b8ca9b036.png)
![Asset](https://user-images.githubusercontent.com/1141514/214654614-92e1b54d-f070-40c4-b226-e8c9ce9e190f.png)

Reply:
I don't think so? IIRC, it's a `DATE` field in the database, and that's just in the `casts` section, not the validation rules. That said, I'm definitely willing to be convinced.

Reply:
I think GitHub's preview might be confusing. Lines 108 and 109 are in `$rules`. What caught my eye is that `Asset` has  `date` included in it's rules (`date|date_format:Y-m-d H:i:s|nullable'`) while the other models only have `date_format` (`date_format:Y-m-d|nullable`) which causes a different validation error message. My screenshots show that the error messages for `Component` returns one message for `purchase_date` while `Asset` returns two messages with the important message being second.

Reply:
I know there was a reason for doing it that way, I just can't remember what it was :( I had tried many variations on it and that's what I came up with, but I can re-test removing it (and be reminded of why I did it that way)